packages/forms-web-app/src/dynamic-forms/validator/string-validator.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars */
+const { body } = require('express-validator');
+const BaseValidator = require('./base-validator.js');
+
+class StringValidator extends BaseValidator {
+	regex = {
+		regex: '',
+		regexMessage: ''
+	};
+	minLength = {
+		minLength: 0,
+		minLengthMessage: ''
```

Comment:
Lets ask about some default error messages at stand up, hopefully there will be some content we can use on one of the tickets

Reply:
Please update the JIRA ticket with any defaults

Reply:
Will do, I'll raise it now